test/livebook/intellisense_test.exs

```diff
@@ -3,22 +3,62 @@ defmodule Livebook.IntellisenseTest do
 
   alias Livebook.Intellisense
 
+  @tmp_dir "tmp/test/remote_intellisense"
+
   # Returns intellisense context resulting from evaluating
   # the given block of code in a fresh context.
-  defmacrop eval(do: block) do
+  defmacrop eval(node \\ nil, do: block) do
     quote do
       block = unquote(Macro.escape(block))
       binding = []
       env = Code.env_for_eval([])
       {value, binding, env} = Code.eval_quoted_with_env(block, binding, env)
+      node = unquote(node) || node()
 
       %{
         env: env,
-        map_binding: fn fun -> fun.(binding) end
+        map_binding: fn fun -> fun.(binding) end,
+        node: node
       }
     end
   end
 
+  setup_all do
```

Comment:
Let's move this to the describe block.

Reply:
I don't think  we can have a setup_all inside the describe block. Therefore my proposal is to keep these tests in a separate file.

Reply:
We could do a separate module in the same file just so it's still grouped together.

Reply:
Separate module or separate file are both fine by me!

Reply:
I moved it back to a separate file